lib/jazzy/sourcekitten.rb

```diff
@@ -45,8 +45,11 @@ def self.make_doc_urls(docs, parents)
 
     # Run sourcekitten with given arguments and return STDOUT
     def self.run_sourcekitten(arguments)
-      bin_path = Pathname(__FILE__).parent + '../../bin'
-      `#{bin_path}/sourcekitten #{(arguments).join(' ')}`
+      bin_path = Pathname(__FILE__).parent + 'sourcekitten/sourcekitten'
```

Comment:
minor gripe: I'm now referring to the project as 'SourceKitten' but the command-line executable as 'sourcekitten', so I'd prefer if we changed to path to `SourceKitten/sourcekitten`